pymemcache/serde.py

```diff
@@ -55,11 +56,11 @@ def python_memcache_deserializer(key, value, flags):
         return int(value)
 
     if flags & FLAG_LONG:
-        return long(value)
+        return long_type(value)
```

Comment:
`long` does not exist in Python 3

In fact even calling `long(x)` is unnecessary on Python 2 since `int`s automatically become `long`s at a certain size.

I'd say drop the `long` flag altogether but obviously that's impossible now since people have serialized and stored data with this code.